tsconfig.cjs.json

```diff
@@ -1,7 +1,6 @@
 {
   "extends": "./tsconfig.json",
   "compilerOptions": {
-    "outDir": "dist/cjs",
-    "declaration": false
```

Comment:
I tried using the same type definition files for both require and import, but then TypeScript thinks that the type definitions are for ESM even in CJS mode, so it won't let you import it in a CJS app.